app/models/post.rb

```diff
@@ -1,20 +1,23 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: :author_id, dependent: :destroy
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   has_many :likes, dependent: :destroy
-  has_many :coments, dependent: :destroy
+  has_many :comments, dependent: :destroy # Fix the typo here
 
-  validates :title, presence: true
+  validates :title, presence: true, length: { maximum: 100 }
```

Comment:
- [ ] [REQUIRED] Great job validating the `Post` model, however, the Project requirement states that 
title must not exceed 250 characters. Kindly resolve this as you are validating against 100 characters at the moment.